events/tests/test_event_api.py

```diff
@@ -283,3 +298,31 @@ def test_event_cannot_be_created_in_approved_state(api_client, contract_zone):
 
     new_event = Event.objects.latest("id")
     assert new_event.state == Event.WAITING_FOR_APPROVAL
+
+
+def test_event_cannot_be_created_when_days_are_full(
+    official_api_client, contract_zone, settings
+):
+    events = EventFactory.create_batch(
+        3, start_time=EVENT_DATA["start_time"], end_time=EVENT_DATA["end_time"]
+    )
+    assert all(event.contract_zone == contract_zone for event in events)
+
+    ret = post(official_api_client, LIST_URL, EVENT_DATA, 400)
```

Comment:
what does `ret` stand for?

Reply:
not sure, but it was used earlier in the same file as well :D

Reply:
should it be `res` at least? : D

Reply:
I think it was intentionally not `res` / `response`. ANYWAY, changed it to `response_data` now :D

Reply:
I am not against `ret`, just was curious what was meant by it. ANYWAY, looks good now : D

Reply:
I am, it was stupid.